android/guava-tests/benchmark/com/google/common/hash/ChecksumBenchmark.java

```diff
@@ -111,6 +111,23 @@ byte fingerprintChecksum(int reps) throws Exception {
     return result;
   }
 
+  // HighwayFingerprint64
+
+  @Benchmark
+  byte highwayFingerprint64HashFunction(int reps) {
+    return runHashFunction(reps, Hashing.highwayFingerprint64());
```

Comment:
This method does not exist in the Hashing class, or maybe it's not accessible due to scope or visibility issues